CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/value/operations/And.java

```diff
@@ -25,7 +25,7 @@
  */
 public class And {
 
-    public static final String NAME = "AND";
+    public static final String NAME = new String("AND");
```

Comment:
What's the reason for using the constructor here (and other related files)?

Reply:
So for some reason, the map structure that this gets stored into, does not work when you omit the new String(...) constructor. I previously fixed this issue up in the SQ warnings from a few weeks back - but this specifically broke the Layers View. It was pretty hard to debug.
I've reverted it back which fixed the issue.

Reply:
hmmm that's really odd. Perhaps the constructor actually behaves differently than expected (though I don't understand why)